src/TTCache.php

```diff
@@ -31,20 +32,26 @@ class TTCache
 
     protected TaggedStore $cache;
 
-    public function __construct(CacheInterface $cache, Closure $keyHasher = null)
+    /**
+     * @var string
+     */
+    private string $specialKeyDelimeter;
```

Comment:
Not convinced there's a lot of value in having this be configurable. How about we just make use of `-` instead of `:`  and get on with things instead.